third-culture-software/bhima
# Improve unpaid invoices report

Improve UI for Unpaid Invoices report

Closes https://github.com/IMA-WorldHealth/bhima/issues/6623

**TESTING**
- Use bhima_test
- Go to:  Finances > Report > Unpaid Invoices Report
  - Choose dates: "This year"
  - Do not select a service, note the item listed
  - Repeat and select the service "Test Service", note that the report title block shows "Test Service"
  - Repeat and select another service.  Even though there are not items found, the service name
    will be shown in the title block.  This makes the report clearer.
